### PR TITLE
feat: fix NDCGEvaluator plus add docs and readme

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

File: evaluators/rank/NdcgEvaluator/README.md [hunk 1]
```diff
@@ -1,4 +1,67 @@
...
+A simple example on how one would use it in Python.
+
+```python
+expected = [('1', 5.0), ('2', 3.0), ('3', 4.0)]
+results = [('2', 1.0), ('1', 0.8), ('3', 0.7)]
+evaluator = NDCGEvaluator(eval_at=5, power_relevance=False)
+evaluation = evaluator.evaluate(actual=results, desired=expected)
+```
+
+This is how it would look a yaml configuration to be loaded inside a Jina Flow.
```
Comment (cristianmtr) [inline evaluators/rank/NdcgEvaluator/README.md:17]:
```suggestion
Example of a YAML configuration:
```

**Review by cristianmtr — APPROVED** [review]
just a minor thing

**Comment by JoanFM** [comment]
@bwanglzu this is the PR for you to add the fix we discussed

File: evaluators/rank/NdcgEvaluator/README.md [hunk 2]
```diff
@@ -1,4 +1,67 @@
...
+
+## Usage example.
+A simple example on how one would use it in Python.
+
+```python
+expected = [('1', 5.0), ('2', 3.0), ('3', 4.0)]
+results = [('2', 1.0), ('1', 0.8), ('3', 0.7)]
+evaluator = NDCGEvaluator(eval_at=5, power_relevance=False)
+evaluation = evaluator.evaluate(actual=results, desired=expected)
+```
```
Comment (bwanglzu) [inline evaluators/rank/NdcgEvaluator/README.md:15]: maybe we can introduce a bit on `power_relevance` here

Comment (JoanFM) [reply evaluators/rank/NdcgEvaluator/README.md:15]: Okey, u can do it as part of the fix if u want

**Comment by bwanglzu** [comment]
image

File: evaluators/rank/NdcgEvaluator/README.md (hunk 2 above)
Comment (JoanFM) [reply evaluators/rank/NdcgEvaluator/README.md:25]: I explained above

File: evaluators/rank/NdcgEvaluator/manifest.yml [hunk 3]
```diff
@@ -2,11 +2,11 @@ manifest_version: 1
 name: NDCGEvaluator
 kind: evaluator
 type: pod
-description: evaluates normalized discounted cumulative gain for information retrieval.
+description: Evaluates normalized discounted cumulative gain for information retrieval.
 author: Jina AI Dev-Team (dev-team@jina.ai)
 url: https://jina.ai
 vendor: Jina AI Limited
 documentation: https://github.com/jina-ai/jina-hub
```
Comment (deepankarm) [inline evaluators/rank/NdcgEvaluator/manifest.yml:9]:
```suggestion
documentation: https://github.com/jina-ai/jina-hub/blob/master/evaluators/rank/NdcgEvaluator/README.md
```

**Review by bwanglzu — APPROVED** [review]
LGTM!

**Pull request merged**